visualization/src/pcl_visualizer.cpp

```diff
@@ -1904,40 +1910,40 @@ pcl::visualization::PCLVisualizer::initCameraParameters ()
   setCameraParameters (camera_temp);
 }
 
-/////////////////////////////////////////////////////////////////////////////////////////////
+
 bool
-pcl::visualization::PCLVisualizer::cameraParamsSet () const
+PCLVisualizer::cameraParamsSet () const
 {
   return (camera_set_);
 }
 
-/////////////////////////////////////////////////////////////////////////////////////////////
+
 bool
-pcl::visualization::PCLVisualizer::cameraFileLoaded () const
+PCLVisualizer::cameraFileLoaded () const
 {
   return (camera_file_loaded_);
 }
 
-/////////////////////////////////////////////////////////////////////////////////////////////
+
 std::string
-pcl::visualization::PCLVisualizer::getCameraFile () const
+PCLVisualizer::getCameraFile () const
 {
   return (style_->getCameraFile ());
 }
 
-/////////////////////////////////////////////////////////////////////////////////////////////
+
 void
-pcl::visualization::PCLVisualizer::updateCamera ()
+PCL_DEPRECATED(1, 12, "pcl::visualization::PCLVisualizer::updateCamera is deprecated, just re-rendering all scenes now.")
```

Comment:
Deprecation needs to come before the return type. [Syntax](https://en.cppreference.com/w/cpp/language/attributes/deprecated)

Reply:
Solved in 9fa4c161979027a0f3385e5ef163c33af1155276 in PR #3925